### DISCUSSION TITLE
Cannot connect to router

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#45] [Q&A]
After updating to the latest version ([Leave Blank v23.05.3 re1), i can connect to the router over 192.168.1.1.
I only setup my password and copied the network, firewall and dhcp files to the NUC.
After a restart i cannot reach Luci anymore (error: ERR_CONNECTION_REFUSED). 

Didn't have this on [Leave Blank v23.05.0). How can this be solved?
Strange thing is I have a working internetconnection, only the router cannot be reached.
Tried both https://192.168.1.1 and http://192.168.1.1

**Comment by tommiefr** [comment]
Aynu update? Or am I the only one?

**Comment by nicholas-opensource** [comment]
Try to restart uhttpd via SSH, some actions probably created some conflict
